tests/before_script.sh

```diff
@@ -21,16 +21,16 @@ sudo start mysql
 
 # setup GLPI and its plugins
 mysql -u root -e 'create database $DBNAME;'
-git clone --depth=1 $GLPI_SOURCE -b $GLPI_BRANCH ../glpi && cd ../glpi
+git clone --depth=35 $GLPI_SOURCE -b $GLPI_BRANCH ../glpi && cd ../glpi
 composer install --no-dev --no-interaction
 if [ -e scripts/cliinstall.php ] ; then php scripts/cliinstall.php --db=glpitest --user=root --tests ; fi
 if [ -e tools/cliinstall.php ] ; then php tools/cliinstall.php --db=glpitest --user=root --tests ; fi
-mkdir plugins/fusioninventory && git clone --depth=1 $FI_SOURCE -b $FI_BRANCH plugins/fusioninventory
+mkdir plugins/fusioninventory && git clone --depth=35 $FI_SOURCE -b $FI_BRANCH plugins/fusioninventory
 IFS=/ read -a repo <<< $TRAVIS_REPO_SLUG
 mv ../${repo[1]} plugins/flyvemdm
 cd plugins/fusioninventory
 if [[ $FI_BRANCH == "glpi9.2+1.0" ]] ; then patch -p1 < ../flyvemdm/tests/patches/fi-fix-obsolete-query.patch; fi
```

Comment:
is this still needed ? You changed the tag of FI to 9.2+2.0 !

Reply:
we have a matrix with GLPI 9.2.1 and FI 9.2+1.0